DNA Sequence Comparison Code.py

```diff
@@ -1,46 +1,67 @@
 from Bio import SeqIO
-from Bio import pairwise2
+from Bio.Align import PairwiseAligner
+from Bio.Seq import Seq
+
 multi_seqfn = "./data/project_dog_dna/dog_breeds.fa"
 mystery_seqfn = "./data/project_dog_dna/mystery.fa"
 
 def DNA_Seq_ID(multi_seqfn: str) -> dict:
     """ Reads in FASTA file and returns a dictionary of sequences.
 
+        Description
+
+        Parameters -
+        Returns - 
 
-        Example:"""
+        Example:
+    """
     sequences = {}
+
     with open(multi_seqfn, 'r') as f:
         for record in SeqIO.parse(f, "fasta"):
             sequences[record.id] = str(record.seq) #uses biopython to iterate over each line in the file and store the sequences in a dictionary
     return sequences
 
-def find_best_match(mystery_seqfn: str, sequences: dict) -> tuple:
+def find_best_match(mystery_seq: str, sequences: dict) -> tuple:
     """ Find the closest sequence in the multi-sequence file to the mystery sequence.
 
         Takes the dictionary from FASTA file conversion dictionary
+
+        Parameters - 
+        Returns - 
+
+        Example:
     """
+    Aligner = PairwiseAligner()
```

Comment:
It is a convention that Python objects, as opposed to classes, are named in lower case. It is best to follow this, but whatever name you use for variables, use them consistently.

One reason why objects should be named in lower case is that usually Python classes are capitalised, and so naming objects, which are simply instances of classes, by capitalising them can create some confusion on whether they represent objects or classes.